nestkernel/Makefile.am

```diff
@@ -37,7 +37,7 @@ libnest_la_SOURCES=\
 		communicator.h communicator_impl.h communicator.cpp\
 		sibling_container.h sibling_container.cpp\
 		subnet.h subnet.cpp\
-		connection.h\
+		connection.h connection_label.h\
```

Comment:
Put `connection_label.h` on a separate line. Only h, cpp and impl files belonging together should be combined on a single line.